Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_controller.hpp

```diff
@@ -11,50 +11,44 @@
 #include "spdlog/spdlog.h"
 #include "../setup_helper.hpp"
 
+/// <summary>
+/// This class is designed as the object to control all object recognition networks and define corresponding functions 
+/// </summary>
 class MLController {
 
+	// The following two lines are for testing and demonstration, not used in the actual system
```

Comment:
This can be removed (and the commented out code), we changed from separate interfaces based on frame type to just a general ModelInterface as is in line 24.